automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala

```diff
@@ -280,6 +282,64 @@ trait Orchestration extends FireCloudClient with LazyLogging {
     postRequestWithMultipart(apiUrl(s"api/workspaces/$ns/$wsName/importEntities"), fileName, fileContent)
   }
 
+  object trial {
+
+    case class TrialProjectReport(unverified: Int,
```

Comment:
Consider renaming the case class as `TrialProjects` and its instantiations as `trialProjects` so it later on reads like `trialProjects.available`, etc.